src/lib/multipageloader.cc

```diff
@@ -254,6 +254,7 @@ ResourceObject::ResourceObject(MultiPageLoaderPrivate & mpl, const QUrl & u, con
 	webPage.mainFrame()->setZoomFactor(devicePixelRatio * settings.zoomFactor); // Zoom in the page to achieve a higher DPI.
 	webPage.setDevicePixelRatio(devicePixelRatio); // Fix CSS media queries (does not affect anything else).
 #endif
+	webPage.mainFrame()->view()->setMediaType(settings.printMediaType ? "print" : "screen");
```

Comment:
This should probably move up a line to fix the compilation failure? I think earlier you overrode the method, any particular reason this was changed?

Reply:
Sorry about that, I've pushed a wrong commit this time!